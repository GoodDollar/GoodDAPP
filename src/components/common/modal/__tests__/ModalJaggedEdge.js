import React from 'react'
import renderer from 'react-test-renderer'
import ImportedModalJaggedEdge from '../ModalJaggedEdge'
import { withThemeProvider } from '../../../../__tests__/__util__'
const ModalJaggedEdge = withThemeProvider(ImportedModalJaggedEdge)

// Note: test renderer must be required after react-native.

describe('ModalJaggedEdge', () => {
  it('matches snapshot', async () => {
    let component

    // eslint-disable-next-line require-await
    await renderer.act(async () => (component = renderer.create(<ModalJaggedEdge />)))
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
