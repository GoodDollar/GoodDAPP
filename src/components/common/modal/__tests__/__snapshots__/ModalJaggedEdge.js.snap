// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ModalJaggedEdge matches snapshot 1`] = `
<div
  className="css-view-1dbjc4n r-flex-13awgt0 r-pointerEvents-12vffkv"
>
  <div
    className="css-view-1dbjc4n r-backgroundImage-693lkh r-backgroundPosition-psfvt2 r-backgroundRepeat-f6vetv r-backgroundSize-jug86b r-height-10ptun7 r-position-bnwqim r-width-13qz1uu r-zIndex-pezta"
  />
</div>
`;
